### DISCUSSION TITLE
mdx-bundler

### DISCUSSION THREAD

**Discussion by joshmedeski** [epicweb-dev/epic-stack#143] [Ideas]
Hi @kentcdodds, I know `mdx` is a big part of your personal website, and I like what Girish did with the [speed-metal-stack](https://github.com/Girish21/speed-metal-stack).

Any plans on adding your [mdx-bundler](https://github.com/kentcdodds/mdx-bundler) to the Epic Stack? I'd be happy to contribute. It would be a great starting point for a blog, documentation, etc... which I could see myself reaching for in any future project.

**Comment by kentcdodds** [comment]
I don't think we'll build it in because it's not useful for the kind of products I think most people will build. We'll eventually do a proper CMS. But I think an example would be very helpful. Would you like to make one?

**Reply by joshmedeski** [reply]
Yeah, I'd like to build my own stack which would include MDX support. I'll be taking lots of inspiration from Epic Stack 😄

**Reply by kentcdodds** [reply]
Terrific! Here are instructions on how to create an Epic Stack example: https://github.com/epicweb-dev/epic-stack/blob/main/docs/examples.md
